paypack/driver/api/transaction.go

```diff
@@ -0,0 +1,118 @@
+package api
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/quarksgroup/paypack-go/paypack"
+)
+
+type transactionService struct {
+	http *wrapper
+}
+
+func (s *transactionService) Cashin(ctx context.Context, tx *paypack.TxPayload) (*paypack.TxResponse, error) {
+	endpoint := "transactions/cashin"
```

Comment:
Make this variable and others like it constants: `const endpoint = "transaction/cashin"`